app/controllers/admin/badge_achievements_controller.rb

```diff
@@ -0,0 +1,44 @@
+module Admin
+  class BadgeAchievementsController < Admin::ApplicationController
+    layout "admin"
+
+    def index
+      @badge_achievements = BadgeAchievement.all.page(params[:page]).order(created_at: :desc).per(10)
```

Comment:
Using `.all` here is not going to be practical since we have 354k of these in the DEV database.